include/galsim/SBMoffatImpl.h

```diff
@@ -45,26 +45,26 @@ namespace galsim {
         double maxK() const;
         double stepK() const;
 
-        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const 
+        void getXRange(double& xmin, double& xmax, std::vector<double>& ) const
```

Comment:
Do you know why your PRs always include all these white space changes?  It would be helpful if you could somehow avoid committing these changes.  It makes it hard to to the review when I have to verify that most of these line changes just involve a single space at the end of a line being removed, which isn't relevant to anything.

You can use `git add -p` to pick through the local changes and select only a subset of them to commit.

Also, if you want to make the white space changes, you can commit them on master and then merge them into the issue branch, so they won't show up on the pull review.  White space changes like this don't have to be reviewed..  :)